### PR TITLE
minor updates to the nRF armcc target

### PR DESCRIPTION
**Pull request by rgrover**
CR: @autopulated

### CODE REVIEW

File: nordic-nrf51822-16k-armcc/CMake/toolchain.cmake [hunk 1]
```diff
@@ -33,11 +33,13 @@ function(yotta_apply_target_rules target_type target_name)
...
         add_custom_command(TARGET ${target_name}
             POST_BUILD
+            COMMAND arm-none-eabi-size ${target_name}
             # fromelf to hex
-            COMMAND fromelf --i32 --output=${target_name}.hex ${target_name}
+            COMMAND fromelf --i32combined --output=${target_name}.hex ${target_name}
             # and append the softdevice hex file
             COMMAND srec_cat ${NRF51822_SOFTDEVICE_HEX_FILE} -intel ${target_name}.hex -intel -o ${target_name}-combined.hex -intel --line-length=44
             COMMENT "hexifying and adding softdevice to ${target_name}"
+            COMMAND srec_info ${target_name}-combined.hex -intel
```
Comment (autopulated) [inline nordic-nrf51822-16k-armcc/CMake/toolchain.cmake]: this COMMAND line should probably go before COMMENT?

Comment (rgrover) [inline nordic-nrf51822-16k-armcc/CMake/toolchain.cmake]: yes, thanks. I'll update this.

**Comment by autopulated** [comment]
looks good to me

**Comment by autopulated** [comment]
I just bumped the version and published

**Pull request merged**